caip10.md

```diff
@@ -0,0 +1,113 @@
+---
+namespace-identifier: icp-caip10
+title: ICP Namespace - Chains
+author: Quint Daenen (@q-uint)
+discussions-to: TODO
+status: Draft
+type: Standard
+created: 2023-05-24
+requires: [ CAIP-2, CAIP-10 ]
+---
+
+# CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
```

Comment:
No, I think the PR is titled CAIP-10 because it the spec proposed follows CAIP-10: https://github.com/ChainAgnostic/CAIPs/blob/master/CAIPs/caip-10.md